deps/rabbitmq_stream_management/src/rabbit_stream_mgmt_db.erl

```diff
@@ -177,3 +177,13 @@ format_resource(_, unknown) ->
 format_resource(NameAs,
                 #resource{name = Name, virtual_host = VHost}) ->
     [{NameAs, Name}, {vhost, VHost}].
+
+ets_select(T, Spec) ->
+    try
+        ets:select(T, Spec)
+    catch error:Reason ->
+              %% badarg can occur if the table has no been created yet
+              rabbit_log:info("Error while querying ETS table '~tp': ~tp",
```

Comment:
Should this be a warning? An error sounds too harsh (this is a temporary harmless condition) but this is not an info message per se.